v4l2/media.py

```diff
@@ -336,8 +337,14 @@ def interfaces(self):
     def find_id(self, id) -> MediaObject | None:
         return next((o for o in self.objects if o.id == id), None)
 
-    def find_entity(self, name):
+    def find_entity(self, name=None, regex=None):
         for e in self.entities:
-            if fnmatch.fnmatch(e.name, name):
-                return e
+            if name is not None and not fnmatch.fnmatch(e.name, name):
+                continue
+
+            if regex is not None and re.match(regex, e.name) is None:
+                continue
+
+            return e
+
```

Comment:
I think we should just go for a single parameter, regex (assuming we need that feature). But that can be done later, as it requires changing all call sites.